bounties_api/notifications/email.py

```diff
@@ -45,9 +46,9 @@ def render_category(c):
 
     @staticmethod
     def rating_color(rating):
-        if rating >= 0.8:
+        if rating >= 4:
```

Comment:
These were a carry-over from the front end. Not sure we need to check against decimal based ratings at all? cc @c-o-l-o-r

Reply:
@codeluggage yes this is correct because you're only concerned with ratings which are 1-5.